lua_modules/http/httpserver.md

```diff
@@ -0,0 +1,61 @@
+# HTTP Server Module
+| Since  | Origin / Contributor  | Maintainer  | Source  |
+| :----- | :-------------------- | :---------- | :------ |
+| 2015-01-19 | [Vladimir Dronnikov](https://github.com/dvv) | [Vladimir Dronnikov](https://github.com/dvv) | [http.lua](../../lua_modules/http/httpserver.lua) |
+
+This Lua module provides a simple callback implementation of a [HTTP 1.1](https://www.w3.org/Protocols/rfc2616/rfc2616.html) server.
+
+!!!note
+	copy this module to nodemcu by yourself 
+
+
+### Require
+```lua
+httpserver = require("httpserver")
+```
+
+### Release
+```lua
+httpserver = nil
+package.loaded["httpserver"] = nil
+```
+
+## httpserver.createServer()
+Function to start HTTP server.
+
+#### Syntax
+`httpserver.createServer(port, handler(req, res))`
+
+#### Parameters
+- `port`: Port number for HTTP server. Most HTTP servers listen at port 80.
+- `handler`: callback function for when HTTP request was made.
+
+#### Returns
+`net.server` sub module.
+
+#### Notes
+Callback function has 2 arguments: `req` (request) and `res` (response). The first object holds values:
+
+- `conn`: `net.socket` sub module.  **DO NOT** call `:on` or `:send` on this
+  object.
+- `method`: Request method that was used (e.g.`POST` or `GET`)
+- `url`: Requested URL
+- `onheader`: assign a function to this value which will be called as soon as HTTP headers like `content-type` are available.
```

Comment:
Assigning into the table is strange.  If that was my suggestion, I apologize.  Perhaps the callback should *return* both `onheader` and `ondata`?